.gitignore

```diff
@@ -0,0 +1,5 @@
+*.a
+*.o
+*.swp
```

Comment:
please add 
```
*_test
*_bench
*_stress

.idea/
.vscode/
```

Reply:
sure. btw it is recommended to build in a standalone directory (e.g `mkdir -p build; cd build; cmake ..; make`) so I ignore the build/ directory only.